src/libraries/Microsoft.PowerFx.Connectors/EngineExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.IR.Nodes;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors
+{
+    public static class EngineExtensions
+    {
+        public static void EnableTabularConnectors(this Engine engine)
```

Comment:
>EnableTabularConnectors [](http://example.com/codeflow?start=27&length=23)

To support tabular connectors, we need to use an IR transform to inject the ServiceProvider at runtime (which contains the HttpClient)